sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -705,7 +705,7 @@ protected void finalize() throws Throwable {
 
   @Override
   public boolean onKey(View view, int i, KeyEvent keyEvent) {
-    if(keyEvent.getKeyCode() == KeyEvent.KEYCODE_BACK) {
+    if(keyEvent.getKeyCode() == KeyEvent.KEYCODE_BACK && _player.getVRMode() == VrMode.STEREO) {
       switchVRMode(VrMode.MONO);
```

Comment:
What do you think about this approach in `switchVRMode`:
`if (_player.getVRMode() == mode) {return;}`